src/components/Forms/Register-Customer.js

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from 'react';
+
+class RegisterCustomer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fName: '',
+      lName: '',
+      email: '',
+      phone: '',
+      zipcode: '',
+      address: '',
+      city: '',
+      state: '',
+      country: '',
+      photo_url: '',
```

Comment:
Since we're initializing our state here, this would be a great spot to have 'photo_url' set to a default profile image url instead of an empty string.